source/includes/kubernetes/_k8_storageclasses.md

```diff
@@ -0,0 +1,139 @@
+        "uid": "f289c0e6-3f20-4274-8cb8-5db8b34dece6"
+      },
+      "parameters": {
+        "clusterID": "rook-ceph",
+        "csi.storage.k8s.io/controller-expand-secret-name": "rook-csi-rbd-provisioner",
+        "csi.storage.k8s.io/controller-expand-secret-namespace": "rook-ceph",
+        "csi.storage.k8s.io/fstype": "ext4",
+        "csi.storage.k8s.io/node-stage-secret-name": "rook-csi-rbd-node",
+        "csi.storage.k8s.io/node-stage-secret-namespace": "rook-ceph",
+        "csi.storage.k8s.io/provisioner-secret-name": "rook-csi-rbd-provisioner",
+        "csi.storage.k8s.io/provisioner-secret-namespace": "rook-ceph",
+        "imageFeatures": "layering",
+        "imageFormat": "2",
+        "pool": "replicapool"
+      },
+      "provisioner": "rook-ceph.rbd.csi.ceph.com",
+      "reclaimPolicy": "Delete",
+      "volumeBindingMode": "Immediate"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/storageclasses?cluster_id=:cluster_id</code>
+
+Retrieve a list of all storage classes in a given [environment](#administration-environments).
+
+| Required                   | &nbsp;                                                      |
+| -------------------------- | ----------------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to list the storage classes. |
+
+| Attributes                           | &nbsp;                                                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id` <br/>_string_                   | The id of the storage class. This is the name of the resource                                                                               |
+| `isDefault` <br/>_boolean_           | Whether or not the storage class is the default one                                                                                         |
+| `allowVolumExpansion` <br/>_boolean_ | Whether not the storage class allows for expandeable volumes                                                                                |
+| `metadata` <br/>_object_             | The metadata of the storage class.                                                                                                          |
+| `parameters` <br/>_object_           | The parameters for the storage provisioner. These are storage provisioner specific and you will likely have to read external documentation. |
+| `provisioner` <br/>_string_          | The provsioner for the storage class                                                                                                        |
```

Comment:
Should be **provisioner** (also in 3 other places)

Reply:
9a00789adfe29db28eeb51a43b40276aad803893